bool status_server
std_msgs/Float64MultiArray data
